duckduckgo/Android
# Move ic_image_24 to common-ui so it can be used elsewhere (also, fixes alpha value for it)

Task/Issue URL: https://app.asana.com/0/488551667048375/1206057713910057/f 

### Description

### Steps to test this PR

Optionally, set up sync and ensure the `ic_image_24` image still shows properly next to `Fetch bookmark icons`
